test/hotspot/jtreg/runtime/Safepoint/TestAbortOnVMOperationTimeout.java

```diff
@@ -51,9 +51,9 @@ public static void main(String[] args) throws Exception {
             return;
         }
 
-        // These should definitely pass: more than a minute is enough for Serial to act.
+        // These should definitely pass: more than 3 minutes is enough for Serial to act.
         // The values are deliberately non-round to trip off periodic task granularity.
-        for (int delay : new int[]{63423, 12388131}) {
+        for (int delay : new int[]{183423}) {
             testWith(delay, true);
         }
```

Comment:
With only one value there is no point having the loop. Why not just replace the shorter, failing, delay with the new longer one?

Reply:
> With only one value there is no point having the loop

Good caught. I just obey the previous code style and forget to refactor it. Will fix it.

> Why not just replace the shorter, failing, delay with the new longer one?

The longer one is useless actually. If the shorter one, `63423` previously, passed,
it means the gc time is less than `63423`. Then the longer one will always pass.
And jtreg would report `time out` in seraval minutes normally,
so the longer time `12388131` is useless.
So I want to remove the longer one and just remain the shorter one.